release.nix

```diff
@@ -5,6 +5,15 @@
     # TODO: change the version number to the actual lorri versions, (and transfer the changelog to a real conventional changelog file, then update lorri self-upgrade (maybe use toml?))
     # Find the current version number with `git log --pretty=%h | wc -l`
     entries = [
+      {
+        version = 872;
+        changes = ''
+          Add support for builtins.filterSource
```

Comment:
This is also missing the first commit, not watching `readDir` recursively by default is a huge win.

Reply:
added it